scripts/sync_check/boot_service.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# The .env file contains environment variables that we want access to.
+# The .env file contains the chain variables that we want access to.
```

Comment:
I think you accidentally did a search&replace across the entire repository. You shouldn't be modifying this file.